### PR TITLE
Hotfix/donation page

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://trello.com/c/mvhx90CS/3004-payment-and-landing-pages

- [x] Add donate link to top nav (CTA style will come later)
- [x] Add donation page content
- [x] Test donation functionality
- [x] Remove any old, unused donation-related code

https://user-images.githubusercontent.com/7017118/190119618-027539ad-fa37-477b-9918-6d6b335aba65.mp4

### CODE REVIEW

**Comment by VirginiaDooley** [comment]
I'm following this documentation to add the donate now widget https://developer.paypal.com/sdk/donate/#link-adddonatesdktowebpage. The functionality works, however I want the widget embedded rather than a donate button.

**Comment by VirginiaDooley** [comment]
[Donation Page](https://trello.com/c/Cnh4QwRd/3070-donation-page)

**Comment by pmk01** [comment]
Looks good to me, one point: the final sentence "find out more..." can we make that a new paragraph, put it in bold and and link it to: https://democracyclub.org.uk/about/impact/

**Comment by VirginiaDooley** [comment]
> Looks good to me, one point: the final sentence "find out more..." can we make that a new paragraph, put it in bold and and link it to: https://democracyclub.org.uk/about/impact/

Screen Shot 2022-09-14 at 2 40 01 PM

**Review by pmk01 — APPROVED** [review]
Great! thanks for getting this done.

File: democracy_club/templates/base.html [hunk 1]
```diff
@@ -6,127 +6,130 @@
...
+        <meta name="twitter:card" content="{% block twitter_card_type %}summary{% endblock twitter_card_type %}">
+        <meta name="twitter:site" content="@democlub">
+        <meta name="twitter:image" content="{% block twitter_image %}{{ CANONICAL_URL }}{% static 'images/logo_icon.png' %}{% endblock twitter_image %}" />
+        <meta name="twitter:image:alt" content="{% block twitter_image_alt %}Democracy Club logo{% endblock twitter_image_alt %}">
+        <meta name="twitter:title" content="{% block twitter_title_content %}Democracy Club{% endblock twitter_title_content %}">
+        <meta name="twitter:url" content="{{ CANONICAL_URL }}{{ request.path }}">
+        <meta name="twitter:description" content="{% block twitter_description_content %}We build digital tools to support everyone’s participation in UK democracy.{% endblock twitter_description_content %}"/>
+    {% endblock twitter_tags %}
+    <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Ensures optimal rendering on mobile devices. -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" /> <!-- Optimal Internet Explorer compatibility -->
```
Comment (symroe) [inline democracy_club/templates/base.html:35]: Could you make an issue to add these (if not in already) to the dc utils base template?

**Review by symroe — CHANGES_REQUESTED** [review]
Did you see my [comment in Slack](https://democracyclub.slack.com/archives/C01JEBKF08M/p1663149744902519) about updating the design system? 

I think we can do a quick release with those changes, update the version here and remove the custom CSS

File: democracy_club/templates/base.html (hunk 1 above)
Comment (VirginiaDooley) [reply democracy_club/templates/base.html:35]: Added to issue https://github.com/DemocracyClub/dc_django_utils/issues/30

Comment (VirginiaDooley) [reply democracy_club/templates/base.html:35]: Although I actually it's partially already there. I think I added this when I was trying to get the Paypal SDK to work; I will remove from here.

File: democracy_club/urls.py [hunk 2]
```diff
@@ -121,15 +121,9 @@
...
-    path(
-        "about/funding/",
-        RedirectView.as_view(url=reverse_lazy("about/support_us")),
-        name="funding",
-    ),
     path(
         "support-us/",
-        RedirectView.as_view(url=reverse_lazy("about/support_us")),
-        name="funding",
+        RedirectView.as_view(url=reverse_lazy("about/funding")),
```
Comment (symroe) [inline democracy_club/urls.py:126]: I think this should go to donate?

**Review by symroe — APPROVED** [review]

**Pull request merged**